WooCommerce/Classes/ViewRelated/Coupons/CouponManagementViewController.swift

```diff
@@ -1,25 +1,205 @@
 import UIKit
+import WordPressUI
 
 final class CouponManagementViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView?
```

Comment:
Any reason to have optional `IBOutlet`?

Reply:
Nope, just something which was useful earlier on in writing it but not needed any more. Removed 😃